index.html

```diff
@@ -668,6 +681,110 @@ <h2 id="constraints">Unconstrained Display Surface Selection</h2>
     </section>
 
 
+    <section>
+      <h2 id="changing-display-surfaces">Changing Display Surfaces</h2>
+
+
+      <p>
+        The user agent MUST NOT change the [=display surface=]
+        associated with a [=capture-session=] unless the user has
+        explicitly indicated that they want this change to be
+        performed by interacting with user agent or operating system.
+
+        If the user indicates a change of [=display surface=], the
+        user agent MUST run the following steps:
+      </p>
+
+
+      <ol>
+        <li>
+          <p>If the [=capture-session=] does not have an associated
+            {{CaptureController}}, <var>controller</var>, abort these steps.</p>
+        </li>
+
+
+        <li>
+          <p>If <var>controller.{{CaptureController/[[DisplaySurfaceChangeCallback]]}}</var>
+            is null, abort these steps.</p>
```

Comment:
I think we should do this check later inside a task queued since CaptureController/[DisplaySurfaceChangeCallback] can be changed at any point in time by JS, so the checks should be done within the context event loop.